checker/check-request-property-any-off-updated_test.go

```diff
@@ -0,0 +1,81 @@
+package checker_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/tufin/oasdiff/checker"
+	"github.com/tufin/oasdiff/diff"
+)
+
+// CL: adding 'anyOf' schema to the request body or request body property
+func TestRequestPropertyAnyOfAdded(t *testing.T) {
+	s1, err := open("../data/checker/request_property_any_of_added_base.yaml")
+	require.NoError(t, err)
+	s2, err := open("../data/checker/request_property_any_of_added_revision.yaml")
+	require.NoError(t, err)
+
+	d, osm, err := diff.GetWithOperationsSourcesMap(getConfig(), s1, s2)
+	require.NoError(t, err)
+	errs := checker.CheckBackwardCompatibilityUntilLevel(singleCheckConfig(checker.RequestPropertyAnyOfUpdated), d, osm, checker.INFO)
+
+	require.Len(t, errs, 2)
+
+	require.Equal(t, checker.ApiChange{
```

Comment:
consider using `ElementsMatch` it'll check two arrays are the same without caring about order of elements

Reply:
done, thanks for the suggestion